src/drt/src/dr/FlexDR.cpp

```diff
@@ -502,21 +506,22 @@ void FlexDR::searchRepair(const SearchRepairArgs& args)
   int prev_perc = 0;
   bool isExceed = false;
 
-  vector<unique_ptr<FlexDRWorker>> uworkers;
+  std::vector<std::unique_ptr<FlexDRWorker>> uworkers;
   int batchStepX, batchStepY;
 
   getBatchInfo(batchStepX, batchStepY);
 
-  vector<vector<vector<unique_ptr<FlexDRWorker>>>> workers(batchStepX
-                                                           * batchStepY);
+  std::vector<std::vector<std::vector<std::unique_ptr<FlexDRWorker>>>> workers(
```

Comment:
I think a using std::vector in this cpp would help readability   [which isn't great to start with]

Reply:
I'm going to say I agree, but that feels ill suited to this PR.

This particular case is a good example of the potential for making a large PR larger. The first std::vector in this type is being treated as a virtual 2D array. It's not clear to me that just adding a using declaration is enough to fix the readability issues. I think it's best to keep this PR focused on removing `using namespace std;`, and try evolving readability separately